spec/migrations/20140218232357_add_group_and_user_columns_to_miq_widget_contents_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_migration
+
+describe AddGroupAndUserColumnsToMiqWidgetContents do
+  shared_examples "removing widget contents" do
+    let(:miq_widget_content_stub) { migration_stub(:MiqWidgetContent) }
+
+    it "removes all existing widget content" do
+      miq_widget_content_stub.create!
+      miq_widget_content_stub.create!
```

Comment:
Same as prior PR, can we create just one?  I don't know that testing 1 vs. 2 of the same thing matters.  Maybe I'm misunderstanding?